src/main/java/commands/Min_by_coordinates.java

```diff
@@ -0,0 +1,21 @@
+package commands;
+
+import iostream.Receiver;
+import packets.Request;
+
+public class Min_by_coordinates extends Command {
```

Comment:
Don't use underscores in class names, it's ok if the command has underscores but the class doesn't